kitchen.yml

```diff
@@ -4,6 +4,9 @@ driver:
 
 provisioner:
   name: chef_zero
+  chef_omnibus_install_options: chef
+  require_chef_omnibus: 14.13.11
```

Comment:
We shoudl either require, Chef 14 & Chef 15 or just latest.

Reply:
That's the last chef 12 fix I'm willing to look at.

Reply:
I think it's way to early to restrict our code to just chef 14 and 15.   That requirement is a separate issue from this fix however.   If we really want to do that it would make more sense to have a separate PR for that.